src/components/DigitalCredentials/CredentialsStepper.vue

```diff
@@ -252,7 +275,8 @@ export default class CredentialsStepper extends Vue {
   }
 
   async issueCredential (credentialType: DigitalCredentialTypes): Promise<void> {
-    await LegalServices.sendCredentialOffer(this.getIdentifier, credentialType)
+    const { data: issuedCredential } = await LegalServices.sendCredentialOffer(this.getIdentifier, credentialType)
+    this.issuedCredential = issuedCredential
```

Comment:
Is there any chance of `issuedCredential` being undefined (say, if sendCredentialOffer returns null)? That may affect the reactivity of this.issuedCredential.

Reply:
Added a default assignment to `null`

Reply:
Perfect, thanks